sql/core/src/test/scala/org/apache/spark/sql/CharVarcharTestSuite.scala

```diff
@@ -146,6 +146,22 @@ trait CharVarcharTestSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("char/varchar with null value for partitioned columns") {
+    Seq("CHAR(5)", "VARCHAR(5)").foreach { typ =>
```

Comment:
`typ` -> `type`.

Reply:
Hi @dongjoon-hyun, 
this is the 2nd time I see your comment on `type`/`typ` :). 
The `type` is a keyword in scala and usually, we use `typ` or `xxxType` instead.

Reply:
Oh.. My bad. Thank you for saying that, @yaooqinn .